intake_esm/aggregate.py

```diff
@@ -188,13 +197,13 @@ def set_coords(ds, varname):
     return ds.set_coords(coord_vars)
 
 
-def open_dataset(url, data_vars, chunk_size=default_chunk_size):
+def _open_dataset(url, data_vars, chunk_size=config.get('default_chunk_size'), **kwargs):
     """open dataset with chunks determined."""
     with dask.config.set({'array.chunk-size': chunk_size}):
-        ds = xr.open_dataset(url, chunks={'time': 'auto'}, decode_times=False)
+        ds = xr.open_dataset(url, chunks={'time': 'auto'}, **kwargs)
     ds.attrs['history'] = f"{datetime.now()} xarray.open_dataset('{url}')"
 
     return set_coords(ds, data_vars)
 
 
-open_dataset_delayed = dask.delayed(open_dataset)
+open_dataset = dask.delayed(_open_dataset)
```

Comment:
Do we want to preserved a public method for non-delayed access? I don't really understand the performance/robustness issues here. I am happy assuming `dask` in general, but should we insist on `dask.delayed`?

Reply:
> Do we want to preserved a public method for non-delayed access?

I don't think this is giving us that much performance gain, if any.  I've disabled it for the time being